glom/core.py

```diff
@@ -248,6 +248,57 @@ def __str__(self):
         return msg
 
 
+if getattr(__builtins__, '__dict__', None) is not None:
+    # pypy's __builtins__ is a module, as is CPython's REPL, but at
+    # normal execution time it's a dict?
+    __builtins__ = __builtins__.__dict__
+
+
+_BUILTIN_ID_NAME_MAP = dict([(id(v), k)
+                             for k, v in __builtins__.items()])
```

Comment:
maybe a dict comprehension would be marginally more readable here?

Reply:
ehhh it's gonna end up in boltons anyways ;)